pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["sphinx-theme-builder @ https://github.com/pradyunsg/sphinx-theme-builder/archive/d9f620b1a73839728c95b596343595d3952ec8bf.zip"]
+requires = ["sphinx-theme-builder @ https://github.com/pradyunsg/sphinx-theme-builder/archive/87214d0671c943992c05e3db01dca997e156e8d6.zip"]
```

Comment:
isn't it possible to rely on a specific release number instead of a commit hash ? last one available is `0.2.0b2`, that would be easier to maintain.

Reply:
It's the one millionth prime. It's just a habit because for the old default RNG (mersenne twist) large primes were considered good choices for seeds. Here we're seeking stability not randomness so the choice of seed really doesn't matter, *and* the default RNG is no longer mersenne.

So, feel free to change if you want. A colleague of mine likes to use 8675309 as an Easter egg seed, in case you think 42 is too obvious

Reply:
Last version tag is from March, the 2 PRs I made that allow this to work were in May and August

Reply:
How exactly do you mean that using a tag URL would be "easier to maintain" than a hash URL?

Reply:
In one look I can check the version number, I'm force to go to the repo history to get the commits corresponding to a hash. But if it's not available yet then let's go for the hash and We'll change it later down the road.